components/identity.oauth.uma.permission.service/src/test/resources/dbScripts/permission.sql

```diff
@@ -0,0 +1,57 @@
+CREATE TABLE IF NOT EXISTS IDN_RESOURCE (
+  ID                  INTEGER   AUTO_INCREMENT NOT NULL,
+  RESOURCE_ID         VARCHAR(255),
+  RESOURCE_NAME       VARCHAR(30),
+  TIME_CREATED        TIMESTAMP DEFAULT '0',
+  RESOURCE_OWNER_ID    VARCHAR(255),
+  TENANT_ID           INTEGER,
+  PRIMARY KEY (ID)
+);
+
+/*CREATE TABLE IF NOT EXISTS IDN_RESOURCE_META_DATA (
```

Comment:
remove commented script

Reply:
fixed in: https://github.com/wso2-extensions/identity-oauth-uma/pull/22/files#diff-7e26d3dcc1c52c9c0ffbdfe85805445aL13